crates/re_space_view/src/view_property_ui.rs

```diff
@@ -166,9 +164,10 @@ fn view_property_component_ui(
             .show_hierarchical(ui, singleline_list_item_content)
     };
 
-    if let Some(tooltip) = field_info.map(|info| info.documentation) {
-        list_item_response = list_item_response.on_hover_text(tooltip);
-    }
+    list_item_response = list_item_response.on_hover_ui(|ui| {
+        let id = egui::Id::new((archetype_name, field.display_name));
+        ui.markdown_ui(id, field.docstring_md);
```

Comment:
It seems that markdown ui somehow adds a lot of space on top and bottom of the hover ui. Is that intentional? Looks like a bug, orthogonal to this PR

![image](https://github.com/rerun-io/rerun/assets/1220815/1627d951-cbdf-474e-b9dc-f5dc7a8ecb79)


Before:
![image](https://github.com/rerun-io/rerun/assets/1220815/40f272ac-475a-4d56-8347-c90470995033)

Reply:
It's a bug: https://github.com/lampsitter/egui_commonmark/issues/54